lib/ios.js

```diff
@@ -112,7 +112,7 @@ class AuthorizationDbCheck extends DoctorCheck {
     try {
       ({stdout} = await exec('security', ['authorizationdb', 'read', 'system.privilege.taskport']));
     } catch (err) {
-      log.warn(err);
+      log.debug(err);
```

Comment:
why debug if this is an error?

Reply:
I tried it. This was actually warning. Will revert.